projects/plugins/jetpack/composer.json

```diff
@@ -11,37 +11,37 @@
 		"ext-fileinfo": "*",
 		"ext-json": "*",
 		"ext-openssl": "*",
-		"automattic/jetpack-a8c-mc-stats": "1.4.x-dev",
```

Comment:
We'll want to keep the dev version of composer here since this is for the monorepo. Easiest way might be just to checkout the copy from master `git checkout master -- projects/plugins/jetpack/composer.json`